Sources/MapboxNavigation/EndOfRouteViewController.swift

```diff
@@ -74,6 +74,10 @@ class EndOfRouteViewController: UIViewController {
         preferredContentSize.height = height(for: .normal)
         updateInterface()
     }
+    
+    override func viewDidLayoutSubviews() {
```

Comment:
In `height(for:)` the `safeAreaInsets` will have the correct size instead of `0`. It would allow the dynamic change of end of route view height.

Reply:
Please also call `super.viewDidLayoutSubviews()` inside this function.

Reply:
Thanks for the suggestion, I missed that.